db/objects/intervention_costings/R__101_intervention-baseline-assumptions.sql

```diff
@@ -0,0 +1,84 @@
+CREATE
+OR REPLACE view intervention_baseline_assumptions AS WITH fortifiable AS (
```

Comment:
```suggestion
CREATE OR REPLACE VIEW intervention_baseline_assumptions AS 
    WITH fortifiable AS (
```
... and indent the rest accordingly

Reply:
#sqlformmaterdiditthatway :wink:.  I can tweak it though!